ilc/client.plugins.js

```diff
@@ -0,0 +1 @@
+module.exports = [];
```

Comment:
could you pls use abstraction because it is not clear why you decided to name file in this manner I mean with . separator